group-generators/helpers/data-providers/bsc-token-provider/index.ts

```diff
@@ -0,0 +1,118 @@
+import axios from "axios";
+import { BscTokenQueryParam, BscTokenQueryResponse, Token } from "./types";
+import { FetchedData } from "topics/group";
+
+export class BscTokenProvider {
+  url: string;
+  headers: {
+    "Content-Type": string;
+  };
+
+  public constructor() {
+    this.url = "https://rpc.ankr.com/multichain/";
+    this.headers = {
+      "Content-Type": "application/json",
+    };
+  }
+
+  public async getNftHolders({ address }: Token): Promise<FetchedData> {
+    const returnData: FetchedData = {};
+
+    const tokenRequestParams: BscTokenQueryParam = {
+      id: 1,
+      jsonrpc: "2.0",
+      method: "ankr_getNFTHolders",
+      params: {
+        blockchain: "bsc",
+        contractAddress: address,
+        pageSize: 1000,
+        pageToken: "",
+      },
+    };
+
+    do {
+      const data: BscTokenQueryResponse = await this.getHolders(
+        tokenRequestParams
+      );
+
+      console.log(data);
```

Comment:
There are some console.log in the code, it would be nice to remove them :)

Reply:
omg.... so sorry. That mistakes shouldn't have been necessary. 😆